### PR TITLE
docs(button): addded danger variants

### PR DESCRIPTION
**Pull request by aagonzales**
Closes https://github.com/carbon-design-system/carbon-website/issues/1926
Closes https://github.com/carbon-design-system/carbon-website/issues/1983
Closes https://github.com/carbon-design-system/carbon-website/issues/1927

Danger button variants added to the website button usage and style tabs.

### CODE REVIEW

File: src/pages/components/button/usage.mdx [hunk 1]
```diff
@@ -641,6 +641,26 @@ will be styled as primary or ghost buttons.
 </Column>
 </Row>
 
+### Danger button variations
+
+The danger button has three different styles: primary, tertiary and ghost.
+Determining which danger button style to use will depend on the level of
+emphasis you want to give to the danger action. Destructive actions that are
+considering a required or primary step in a workflow should use the primary
```
Comment (janchild) [inline src/pages/components/button/usage.mdx:649]:
```suggestion
considered a required or primary step in a workflow should use the primary
```

File: src/pages/components/button/usage.mdx [hunk 2]
```diff
@@ -641,6 +641,26 @@ will be styled as primary or ghost buttons.
...
+danger button style. However, if a destructive action is just one of several
+actions a user could choose then a lower emphasis style like the tertiary danger
+button or the ghost danger button may be more appropriate.
+
+<Row>
+<Column colLg={8}>
+
+![Danger button variations](images/button_usage_31.png)
+
+<Caption>Examples of the primary, tertiary and ghost danger buttons</Caption>
```
Comment (janchild) [inline src/pages/components/button/usage.mdx:659]:
```suggestion
<Caption>Examples of the primary, tertiary, and ghost danger buttons</Caption>
```

File: src/pages/components/button/usage.mdx [hunk 3]
```diff
@@ -70,7 +70,7 @@ desired action is to take the user to a new page.
 | _Primary_   | For the principal call to action on the page. Primary buttons should only appear once per screen (not including the application header or in a modal dialog). |
 | _Secondary_ | For secondary actions on each page; these can only be used  in conjunction with a primary button.                                                             |
 | _Tertiary_  | For less prominent actions; tertiary buttons can be used in isolation  or paired with a primary button when there are multiple calls to action.               |
-| _Danger_    | For actions that could have destructive effects on the user’s data  (delete, remove, etc.).                                                                   |
+| _Danger_    | For actions that could have destructive effects on the user’s data  (delete, remove, etc.). Danger button has three styles: primary, tertiary, and ghost.     |
```
Comment (janchild) [inline src/pages/components/button/usage.mdx:73]:
```suggestion
| _Danger_    | For actions that could have destructive effects on the user’s data  (for example, delete or remove). Danger button has three styles: primary, tertiary, and ghost.     |
```

**Review by janchild — COMMENTED** [review]
@aagonzales looks great! Just some very minor copyedits.

**Review by laurenmrice — CHANGES_REQUESTED** [review]
**Usage tab**

Modifiers image
-Ghost button tile: There should be `8px` space between `Ghost` and the `trash icon`.

**Comment by aagonzales** [comment]
Ok fixed the ghost button images. Good catch!

**Review by laurenmrice — APPROVED** [review]
Awesome ! 🎉

**Review by dakahn — APPROVED** [review]

File: src/pages/components/button/usage.mdx [hunk 4]
```diff
@@ -641,6 +641,26 @@ will be styled as primary or ghost buttons.
 </Column>
 </Row>
 
+### Danger button variations
+
+The danger button has three different styles: primary, tertiary and ghost.
```
Comment (janchild) [inline src/pages/components/button/usage.mdx:646]:
```suggestion
The danger button has three different styles: primary, tertiary, and ghost.
```

File: src/pages/components/button/usage.mdx [hunk 5]
```diff
@@ -641,6 +641,26 @@ will be styled as primary or ghost buttons.
...
+Determining which danger button style to use will depend on the level of
+emphasis you want to give to the danger action. Destructive actions that are
+considered a required or primary step in a workflow should use the primary
+danger button style. However, if a destructive action is just one of several
+actions a user could choose then a lower emphasis style like the tertiary danger
```
Comment (janchild) [inline src/pages/components/button/usage.mdx:651]:
```suggestion
actions a user could choose from, then a lower emphasis style like the tertiary danger
```

**Review by janchild — APPROVED** [review]
Hi @aagonzales, this is great! I'm approving. There are two small tweaks that I missed the first go around, if you could pls pop those in. Thank you!

**Review by tw15egan — APPROVED** [review]

**Pull request merged**